hydra-cluster/bench/Bench/EndToEnd.hs

```diff
@@ -106,7 +106,13 @@ bench timeoutSeconds workDir dataset@Dataset{clientDatasets} clusterSize =
 
                 putTextLn "Closing the Head"
                 send leader $ input "Close" []
+                -- REVIEW: Why 60 * clusterSize?
```

Comment:
@abailly-iohk you have an idea why?

Reply:
I think the larger the cluster is, the greater chances are that nodes will finish processing their txs out of sync and will have to wait for other nodes to finish posting theirs.